docs/resources/dds_instance.md

```diff
@@ -106,6 +167,9 @@ The following arguments are supported:
 * `mode` - (Required, String, ForceNew) Specifies the mode of the database instance. Changing this creates a new
   instance.
 
+* `configuration` - (Optional, List, ForceNew) Specifies the configuration information. The structure is described below.
```

Comment:
as the `configuration` can not set from API response, it's should add an note in `Import` section.